examples/ConnectNoEncryption/ConnectNoEncryption.ino

```diff
@@ -91,61 +92,92 @@ void printWifiData() {
   Serial.print("IP Address: ");
   Serial.println(ip);
 
-  // print your MAC address:
+  // print the MAC address of your WiFi module:
   byte mac[6];
   WiFi.macAddress(mac);
   Serial.print("MAC address: ");
-  Serial.print(mac[5], HEX);
-  Serial.print(":");
-  Serial.print(mac[4], HEX);
-  Serial.print(":");
-  Serial.print(mac[3], HEX);
-  Serial.print(":");
-  Serial.print(mac[2], HEX);
-  Serial.print(":");
-  Serial.print(mac[1], HEX);
-  Serial.print(":");
-  Serial.println(mac[0], HEX);
+  for (uint8_t i = 0; i < 6; i++) {
+    if (mac[i] < 0x10) {
+      Serial.print("0");
+    }
+    Serial.print(mac[i], HEX);
+    if (i != 5) {
+      Serial.print(":");
+    }
+    else {
```

Comment:
use either
```
}
else
{
```
or 
```
} else {
```

Reply:
it's ok if I use the following coding style for all the examples ?
```
if (condition) {
  // do something
}
```
```
if (condition) {
  // do something
} else {
  // do other things
}
```
```
if (condition) {
  // do something
} else if (other_condition) {
  // do other things
} else {
 // ...
}
```

Reply:
yes, fine